torch/functional.py

```diff
@@ -1,6 +1,10 @@
 import torch
 import torch.nn.functional as F
 from itertools import product
+from .lowrank import pca
+from .lowrank import svd as svd_lowrank
```

Comment:
This exposes `pca` and `svd_lowrank` in two places in the API, for `svd` even with different names (`torch.lowrank.svd` and `torch.functional.svd_lowrank`). This seems undesirable.

Reply:
Or should it be `_lowrank.py`? That prevents exposing other public-looking functions as well.